streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBKeyValueStoreTest.java

```diff
@@ -39,7 +40,7 @@
             Deserializer<K> keyDeser = (Deserializer<K>) context.keyDeserializer();
             Serializer<V> valSer = (Serializer<V>) context.valueSerializer();
             Deserializer<V> valDeser = (Deserializer<V>) context.valueDeserializer();
-            supplier = Stores.create("my-store").withKeys(keySer, keyDeser).withValues(valSer, valDeser).persistent().build();
+            supplier = Stores.create("my-store").withKeys(Serializations.serialization(keySer, keyDeser)).withValues(Serializations.serialization(valSer, valDeser)).persistent().build();
```

Comment:
@miguno Here are the rationale behind this:

1) For `StreamsConfig`, we still keep the serdes separately (i.e. four configs) as they are going to be passed separately to the underlying producer and consumer; and hence the context also expose four functions for accessing serdes.

2) In the Streams DSL, there are also some places like `KStream#to()` where only serializers / deserializers are used. We still maintain it as is.